src/ray/gcs/gcs_server/test/gcs_server_test_util.h

```diff
@@ -68,21 +69,25 @@ struct GcsServerMocker {
       return Status::OK();
     }
 
+    /// WorkerLeaseInterface
     void RequestWorkerLease(
         const ray::TaskSpecification &resource_spec,
         const rpc::ClientCallback<rpc::RequestWorkerLeaseReply> &callback,
         const int64_t backlog_size = -1) override {
+      RAY_LOG(ERROR) << "REQUEST_WORKER_LEASE";
       num_workers_requested += 1;
       callbacks.push_back(callback);
     }
 
+    /// WorkerLeaseInterface
```

Comment:
error comment?

Reply:
that function implements the `WorkerLeaseInterface ` interface